app/actions/handlers.py

```diff
@@ -28,7 +28,11 @@ async def action_auth(integration, action_config: AuthenticateConfig):
         password=action_config.password.get_secret_value(),
     )
 
-    token = await mb_client.get_token()
+    try:
+        token = await mb_client.get_token()
+    except Exception as e:
```

Comment:
We should catch more specific errors here. And consider adding a message to the result that's returned, so it'll show up in the UI.

Reply:
+1